app/services/articles/detect_language.rb

```diff
@@ -0,0 +1,17 @@
+module Articles
+  class DetectLanguage
+    def self.call(article)
+      text = extract_text(article)
+      response = CLD.detect_language(text)
+      response[:code] if response[:reliable]
+    rescue StandardError
+      nil
```

Comment:
Just curious why we do nothing if there is an error. I am probably missing some context.

Reply:
This code just moved, so I'm not 💯 qualified to answer this, but I assume language detection is non-critical, so in case we don't manage to do it, we just move on.